shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,134 @@
+import os
+import numpy as np
+from .read_nii import read_nii
+
+
+def dicom_to_nifti(unsortedDicomDir, niftiPath):
+    """
+    TODO: indicate the purpose of this function
+    :param unsortedDicomDir:
+    :param niftiPath:
+    :return:
+    """
+
+    #os.mkdir(niftiPath)
+    print(unsortedDicomDir)
+    print(niftiPath)
+
+    # Check for which "which" to use depending on OS
+    if os.name == 'nt':  # If OS is Windows
+        which = 'where'
+        copy = 'copy'
+    else:
+        which = 'which'
+        copy = 'cp -r'
+
+    # Make sur dcm2niix is installed
+    if os.system(which + ' dcm2niix') != 0:
+        print('Error: dcm2niix is not installed.')
+
+    # Make sure dcm2bids is installed
+    if os.system(which + ' dcm2bids') != 0:
+        raise Exception('Error: dcm2bids is not installed.')
+
+    # Create bids structure for data
+    participant = ''
+    if os.system('dcm2bids_scaffold -o ' + niftiPath) != 0:
+        raise Exception('Error: dcm2bids_scaffold')
```

Comment:
You should almost never use `os.system()`. It is subject to [shell injection](https://en.wikipedia.org/wiki/Code_injection). Instead, use `subprocess.run(['dcm2bids_scaffold', '-o', 'niftiPath'], check=True)`.

The `check=True` makes it throw an exception like you're doing.